packages/picasso.js/src/core/chart-components/axis/axis-node-builder.js

```diff
@@ -260,9 +300,11 @@ export default function nodeBuilder(isDiscrete) {
           tick,
         });
       };
-
       let labelNodes = [];
-      if (layered && (settings.align === 'top' || settings.align === 'bottom')) {
+      if (settings.arc.isRadial) {
```

Comment:
What happens if the settings does not have arc defined. The old versions does not have arc defined in settings.